appservice/src/intTest/groovy/nu/yona/server/AddDeviceTest.groovy

```diff
@@ -2,108 +2,113 @@ package nu.yona.server
 
 
 import groovy.json.*
-import spock.lang.Shared
 
-class AddDeviceTest extends AbstractAppServiceIntegrationTest {
-
-	@Shared
-	def richardQuin
-
-	def 'Add user Richard'(){
+class AddDeviceTest extends AbstractAppServiceIntegrationTest
+{
+	def 'Set new device request'()
```

Comment:
You removed the test `def 'Get new device request with user secret'()`. I would expect a new test or a test that has the name `Set new device request and get it with user secret`.